assets/training/finetune_acft_image/environments/acft_video_mmtracking/asset.yaml

```diff
@@ -0,0 +1,6 @@
+name: acft-mmtracking-video-gpu
```

Comment:
You would need one more PR for the env. 
@PhaniShekhar - had raised a PR to include the envs to a config.

Reply:
it turns out I don't have write permission to this PR... I'll bother Phani or other IDC folks to write for me :(

Reply:
Here is the PR https://github.com/microsoft/mcr/pull/2892